metaseq/launcher/slurm.py

```diff
@@ -288,61 +331,63 @@ def gen_sbatch_command_and_str(
         "--signal",
         "B:USR1@180",
     ]
+    if array_length is not None:
+        sbatch_cmd += ["--array", f"0-{array_length-1}"]
 
     if args.constraint:
         sbatch_cmd += ["--constraint", args.constraint]
 
     if args.partition:
         sbatch_cmd += ["--partition", args.partition]
-
     if args.reservation:
         sbatch_cmd += ["--reservation", args.reservation]
-
     if args.exclusive:
         sbatch_cmd += ["--exclusive"]
-
-    comment = ""
     if args.comment:
         comment = args.comment
-
-    if args.snapshot_code:
-        comment += (
-            f", OSS Code Location: {oss_destination}"
-            if comment
-            else f"OSS Code Location: {oss_destination}"
-        )
+        if args.snapshot_code:
+            comment += f", OSS Code Location: {oss_destination} Internal Code Location: {internal_destination}"
         sbatch_cmd += ["--comment", comment]
-
+    elif args.snapshot_code:
```

Comment:
Nit: This is redundant with above. As annoying as it is, I preferred the `comment = ""` bit as it was